server/monitor/src/main/java/org/apache/accumulo/monitor/next/Endpoints.java

```diff
@@ -119,15 +121,27 @@ public Map<String,String> getEndpoints(@Context HttpServletRequest request) {
   @Produces(MediaType.APPLICATION_JSON)
   @Description("Returns a list of the resource groups that are in use")
   public Set<String> getResourceGroups() {
-    return monitor.getInformationFetcher().getSummary().getResourceGroups();
+    try {
+      return monitor.getInformationFetcher().getSummary().getResourceGroups();
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
+      throw new ServiceUnavailableException(
+          Response.status(Response.Status.SERVICE_UNAVAILABLE).build(), e);
+    }
```

Comment:
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes seems like 5xx status code would be appropriate. `503 Service Unavailable` seemed appropriate